### PR TITLE
Update to utility scripts to compare FITS files and assemble them

### PR DESCRIPTION
**Pull request by menanteau**
Update utility scripts to compare FITS files and assemble them from scratch

### CODE REVIEW

File: bin/assemble_FITS [hunk 1]
```diff
@@ -17,15 +17,15 @@ compress_choices = [None, 0, 11, 'RICE_1', 'PLIO_1', 21, None,
...
+    # 1. We make a proto-parse use to read in the default yaml configuration
+    # file, Turn off help, so we print all options in response to -h
     conf_parser = argparse.ArgumentParser(add_help=False)
     conf_parser.add_argument("-c", "--config", help="assemble FITS config file")
     args, remaining_argv = conf_parser.parse_known_args()
 
     # If we have -c or --config, then we proceed to read it
     if args.config:
-        conf_defaults = yaml.load(open(args.config))
+        conf_defaults = yaml.safe_load(open(args.config))
```
Comment (timj) [inline bin/assemble_FITS]: Doesn't this leak a file descriptor?

File: bin/write_segment_2FITS [hunk 2]
```diff
@@ -18,7 +18,7 @@ out_name_fz = "{}.fits.fz".format(fname)
 fd = open(fname, 'rb')
```
Comment (timj) [inline bin/write_segment_2FITS]: Use a context manager here.

**Comment by timj** [comment]
A general comment, but if you are writing Python executable scripts that do not have `.py` extensions, you will need to update your travis file to run flake8 on each one explicitly -- otherwise they will be ignored from the checks (and the files aren't compliant from a quick glance).

File: bin/write_segment_2FITS (hunk 2 above)
Comment (menanteau) [reply bin/write_segment_2FITS]: Changed to a context manager 
```with open(fname, 'rb') as fb:```

File: bin/assemble_FITS (hunk 1 above)
Comment (menanteau) [reply bin/assemble_FITS]: Changed to: 

```
with open(args.config) as conf:
          conf_defaults = yaml.safe_load(conf)
```

**Pull request merged**